src/commands/Minion/xpgains.ts

```diff
@@ -30,25 +31,30 @@ export default class extends BotCommand {
 			: undefined;
 
 		const res = await this.client.orm.query(
-			`SELECT "new_user".username, sum(xp) as total_xp
+			`SELECT user_id as user, sum(xp) as total_xp, max(date) as lastDate
 FROM xp_gains
-INNER JOIN "new_users" "new_user" ON xp_gains.user_id = "new_user"."id"
 WHERE date > now() - INTERVAL '1 ${interval}'
 ${skillObj ? `AND skill = '${skillObj.id}'` : ''}
-GROUP BY "new_user".username
-ORDER BY total_xp DESC
+GROUP BY user_id
+ORDER BY total_xp DESC, lastDate asc
 LIMIT 10;`
```

Comment:
uppercase on the SQL operators please. (asc), i see i didnt uppercase the `as` but if you could uppercase that too.